examples/neural_networks/plot_random_neural_network.py

```diff
@@ -0,0 +1,155 @@
+"""
+========================================================================
+Impact of varying hyperparameters alpha and weight_scale on toy datasets
```

Comment:
Yah, "The impact of varying alpha and weight_scale on the decision function" makes more sense.